OneCommunityGlobal/HighestGoodNetworkApp
# Sam applications landing page

# Description
Frontend
Should be accessible to the general public at highestgood.com/collaboration
The application landing page must display a list of all job ads available that people can click on and apply to, this should include an image related to the relevant job
Include a clickable link under the main title to return to the collaboration page on the One Community website:
https://www.onecommunityglobal.org/collaboration/
Add a title-based search bar and dropdown list for users to filter and apply, with further details below
Scale the view to fit the number of featured positions. For instance, if 12 positions are available, display all 12; if 18, display all 18
Fit more ads on the page with a layout of 3 rows of 6 ads each
Enable pagination, allowing users to scroll through pages of job listings based on availability (e.g., page navigation at the bottom: 1 2 3 4 5 >>)
Clicking on any of the ads brings up/ takes us to the page that loads all the jobs available in that category
a. The ads are all general categories and clicking on them should take you to a more specific page of listings
b. This page must have a URL like - https://www.onecommunityglobal.org/collaboration/seeking-insert-job-category (for example seeking-mechanical engineer)


## Related PRS (if any):
This frontend PR is related to the [#1152](https://github.com/OneCommunityGlobal/HGNRest/pull/1152)  backend PR.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as any user
5. go to localhost:3000/collaboration
6. Verify the search and filtering functionality. Can also try Adding jobs in the backend (see backend PR) and check if the job shown up in list of jobs.


https://github.com/user-attachments/assets/61eb7349-7d5e-434e-af51-aec09e1ac95e
